clmm/utils.py

```diff
@@ -40,7 +40,9 @@ def compute_radial_averages(xvals, yvals, xbins, error_model='std/sqrt_n'):
         yerr = binned_statistic(xvals, yvals, statistic='std', bins=xbins)[0]
     elif error_model == 'std/sqrt_n':
         yerr = binned_statistic(xvals, yvals, statistic='std', bins=xbins)[0]
```

Comment:
Maybe we can try this here to avoid that `mean` in `binned_statistic` gives NaN when some bin is empty.
```suggestion
        count = binned_statistic(xvals, yvals, statistic='count', bins=xbins)[0]
        count_zero = count==0
        
```

Reply:
@vitenti - I'd be happy to make some of @shenmingfu  's suggested changes in a co-work if they sound reasonable to you.  Let me know.

Reply:
Yes, it seems reasonable to standardize the output with zero instead of NaN for empty bins components. I just modified this branch accord to @shenmingfu suggestions.